metabase/metabase
# Repro for #29347, #29346

Reproduces https://github.com/metabase/metabase/issues/29347
Reproduces https://github.com/metabase/metabase/issues/29346
Related to https://github.com/metabase/metabase/pull/29359

Reproduction steps for field values remapping issues in dashboards.

How to test:
- Run skipped tests
- Tests that open the filter popover should fail when trying to select a mapped value (because the mapping is not applied)
- Tests that open the dashboard with a value in the url /token should fail because the value is not set correctly in the widget

<img width="1108" alt="Screenshot 2023-03-21 at 15 39 56" src="https://user-images.githubusercontent.com/8542534/226624280-d7e89220-2a02-4ca0-bdaf-2d4283343bf6.png">
